src/stores/country-options.md

```diff
@@ -34,3 +34,20 @@ If needed, clear the **Use system value** checkbox for each setting you want to
 1. In the **Top Destinations** list, select the primary countries that you target for sales.
 
 1. When complete, click <span class="btn">Save Config</span>.
+
+## Set the country options for specific delivery method
+
+You can further configure shipping to specific countries for each available [delivery method]({% link /shipping/delivery.md%}) (UPS, FedEx, etc.)
+
+1. On the _Admin_ sidebar, go to **Stores** > **Settings** > **Configuration**> **Sales** > **Delivery Methods**.
```

Comment:
This should be two steps since they are separate menu structures

```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.

 1. In the left panel, expand **Sales** and choose **Delivery Methods**.
```